processes-services/src/main/java/org/exoplatform/processes/listener/CancelRequestNotificationListener.java

```diff
@@ -18,6 +18,8 @@ public void onEvent(Event<TaskDto, ProjectDto> event) {
     ProjectDto projectDto = event.getData();
     NotificationContext ctx = NotificationContextImpl.cloneInstance();
     ctx.append(NotificationArguments.REQUEST_CREATOR, taskDto.getCreatedBy());
+    ctx.append(NotificationArguments.REQUEST_PROCESS, projectDto.getName());
+    ctx.append(NotificationArguments.REQUEST_TITLE, taskDto.getTitle());
```

Comment:
I see you add parameters in the context of the notification.

So, notifications created BEFORE your modification will not have the same parameters than notification created AFTER your modifications

Can you ensure that notifications created BEFORE will not break the interface (js error for example) at least. In the better case, notifications created before can display a small set of informations even if all parameters are not present

Reply:
we can improve it by adding condition on existing params and display the old message.

Reply:
yes please, if it is possible I prefer to make both possibility works

Reply:
done https://github.com/exoplatform/processes/compare/30fc51145f9633622e02e9791ae48e7f85c0b3ac..4d7f083a359ac61e82f4c7bcf557a3e5cfe91f85